slackminion/bot.py

```diff
@@ -199,6 +200,7 @@ def send_message(self, channel, text, thread=None, reply_broadcast=None, attachm
             reply_broadcast=reply_broadcast,
             attachments=attachments,
             parse=parse,
+            link_names=link_names,
```

Comment:
should we pass it only if it is present in the list of args?

Reply:
I think link_names=1 is the expected behavior, or would we want to make every plugin pass a flag?